src/builders/bundle.ts

```diff
@@ -1,6 +1,6 @@
 import { builtinModules } from "node:module";
 import { mkdir, readFile, writeFile } from "node:fs/promises";
-import { dirname, relative, join, basename, extname, resolve } from "node:path";
+import { dirname, relative, join, basename, extname, resolve } from "pathe";
```

Comment:
Do we really need `pathe`?

Reply:
In #43 it was reported that it helps with separator issueson Windows